molecule/Provisioners/__init__.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright (c) 2015 Cisco Systems
```

Comment:
I've corrected the folder to be lower case, but the PR doesn't seem to show that.